### PR TITLE
feat(action): add optional id in form fields

### PR DESCRIPTION
**Pull request by jbarreau**

### CODE REVIEW

File: src/datasource_toolkit/forestadmin/datasource_toolkit/decorators/action/collections.py [hunk 1]
```diff
@@ -40,9 +41,28 @@ def add_action(self, name: str, action: ActionDict):
...
+            if element.TYPE == ActionFieldType.LAYOUT:
+                element = cast(DynamicLayoutElements, element)
+                if element._component == "Row":
+                    self._validate_id_uniqueness(element._row_subfields, used)  # type: ignore
+            else:
+                element = cast(BaseDynamicField, element)
+                if element.id in used:
+                    raise DatasourceToolkitException(
+                        f"All field must have different 'id'. Conflict come from field '{element.id}"
+                    )
```
Comment (nicolasalexandre9) [inline src/datasource_toolkit/forestadmin/datasource_toolkit/decorators/action/collections.py:63]: missing ' after {element.id} right ?

Comment (jbarreau) [reply src/datasource_toolkit/forestadmin/datasource_toolkit/decorators/action/collections.py:63]: 👍

File: src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/action_values.py [hunk 2]
```diff
@@ -71,8 +71,9 @@ def make_form_data(
         """
         data: Dict[str, Any] = {}
         for key, value in raw_data.items():
-            _fields = list(filter(lambda f: f["label"] == key, fields))  # type: ignore
+            _fields = list(filter(lambda f: f["id"] == key, fields))  # type: ignore
             if len(_fields) != 1:
+                # TODO: here
```
Comment (nicolasalexandre9) [inline src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/action_values.py:76]: What is the todo ?

Comment (nicolasalexandre9) [reply src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/action_values.py:76]: After discussion with @jbarreau this line should be deleted

**Review by nicolasalexandre9 — APPROVED** [review]
LGTM 👍

**Pull request merged**